build/source/engine/ffile_info.f90

```diff
@@ -164,7 +174,7 @@ subroutine ffile_info(nGRU,err,message)
   ! loop through all variables in netcdf file, check to see if everything needed to run the model exists and data_step is correct
   do iNC=1,nVar
```

Comment:
Should the loop be over the number of variables in the file or over the number of variables that you already know that you need? Seems to me I could really slow SUMMA down if I give it a file with a huge number of variables that it would not need. Flipping the loop (i.e. specifically looking for the variables that you actually know you need) may be more of a fixed time.

Reply:
We could do it either way. We loop over the number of variables in the data file in case there are efficiency gains by reading data close together on disk. Note that `ffile_info.f90` is only called once at the start of the simulation.